io.botfather.Botfather.appdata.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2018 The Botfather Developers -->
+<component type="desktop-application">
+	<id>io.botfather.Botfather.desktop</id>
+	<metadata_license>CC0-1.0</metadata_license>
+	<project_license>LicenseRef-proprietary=https://botfather.io/docs/legal/tos/</project_license>
+	<url type="homepage">https://botfather.io</url>
+	<name>Botfather</name>
+	<summary>Universal automation framework</summary>
+	
+	<categories>
```

Comment:
You can remove these, they go in the desktop file.

Reply:
Including the categories?

Reply:
Yes the `.desktop` file has a categories entry, they don't need to be in both files.